### ISSUE TITLE
YTPlayerView embedded in UITableViewCell - Playing Video prevents tableView delegate call

### ISSUE THREAD

**Issue by vixlet-stix** [youtube/youtube-ios-player-helper#288]
**Context**

YTPlayerView is embedded within a UITableViewCell.  The player is then loaded In the tableView:cellForRowAt method of the data source.

The above is working fine. The video is loading and the controls are enabled.

**Issue**

I've implemented the tableView:didEndDisplaying:forRowAt method of the table view delegate to capture when a cell is scrolled from the screen.  If I don't play the video, this method gets called on the cell with the YTPlayerView.  But...

When I play the video and scroll the cell off the screen, the delegate method tableView:didEndDisplaying:forRowAt method  DOES NOT GET CALLED on that cell.  Other cells that do not have the player embedded behave properly.  It's just this one cell.

The problem also occurs if I play the video and then pause before I scroll.

I can confirm the cell is removed from the tableView when the cell scrolls off screen, but for some reason, there's something that's preventing the tableView:didEndDisplaying:forRowAt from being called.

Bottom Line: I'd like to pause a video when the cell scrolls off the screen, and then reset the player when the cell is prepared for reuse.  Right now, if I scroll the cell off the screen, the video will keep playing -- even tho the cell is inaccessible.

**Comment by SagarSathvara** [comment]
I am also suffering from same issue.

**Comment by kudzaiTnexus** [comment]
Any updates on this issue, i am having the same issue

**Comment by tylerlantern** [comment]
Happen to me too.

**Comment by weipinglii** [comment]
Same thing happens in UICollection or IGListKit

**Comment by songwentong** [comment]
Happen to me too.

**Comment by Tawfikk** [comment]
and nobody not fixed this issue ?)

**Comment by villegaskpu** [comment]
someone solved the problem?

**Comment by nastasiupta** [comment]
Any update on this one?  @todd-patterson
